js/apps/account-ui/src/root/Root.tsx

```diff
@@ -37,7 +37,12 @@ const ReferrerLink = () => {
 
 export const Root = () => {
   const { t } = useTranslation();
-  const indexHref = useHref("/");
+  const brandImage = environment.logo ? environment.logo : "logo.svg";
+  const logoUrl = environment.logoUrl ? environment.logoUrl : "/";
+  const internalLogoHref = useHref(logoUrl);
+
+  // User can indicate that he wants an internal URL by starting it with "/"
+  const indexHref = logoUrl.startsWith("/") ? internalLogoHref : logoUrl;
```

Comment:
Do we really need this check? It seems like the result from `useHref()` is always usable.

Reply:
The documentation of `useHref()` isn't clear on that.  As I understand it, its purpose is to generate URLs that are internal to the application and compatible with React Router.  But in this case, the value passed in could be something like `http://www.keycloak.com`.  

Even if the behavior worked today and it just returned the URL unchanged, I wouldn't want to rely on it.